torch/_higher_order_ops/executorch_call_delegate.py

```diff
@@ -74,12 +74,12 @@ def _unwrap_proxy(e):
 # pyre-ignore
 def call_delegate_cpu(lowered_module, *args):
     # FX creates this immutable_dict/list concept. Get rid of this.
-    map_types = {
+    map_types: dict[type, type] = {
         torch.fx.immutable_collections.immutable_dict: dict,
         torch.fx.immutable_collections.immutable_list: list,
     }
     new_args = pytree.tree_map_only(
-        tuple(map_types.keys()),
+        tuple(map_types.keys()),  # type: ignore[arg-type]
```

Comment:
Is the arg type error here easily fixiable? Seems like it might expect a sequence?

Reply:
Opened a new PR to address this:

- #147699